src/olympia/versions/models.py

```diff
@@ -399,7 +399,7 @@ def from_upload(cls, upload, addon, selected_apps, channel, parsed_data=None):
         return version
 
     def license_url(self, impala=False):
-        return reverse('addons.license', args=[self.addon.slug, self.version])
+        return reverse('addons.license', args=[self.addon_id, self.version])
```

Comment:
Not sure about this - we expose this in the API response so we're now returning non-canonical frontend urls

Reply:
reverted in f896d03c704fefc4abb1cf471fdf7955a2a3e4fe, it's not necessary.